cmd/server/main.go

```diff
@@ -125,6 +125,45 @@ func buildCLI() *cli.App {
 				return nil
 			},
 		},
+		{
+			Name:      "render-config",
+			Usage:     "Render server config template",
+			ArgsUsage: " ",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
```

Comment:
Can this just use the flags (config,env,zone) from the top-level app instead of redefining them?